EngineLayer/Psm.cs

```diff
@@ -401,14 +401,14 @@ internal void Add(Psm psmParent)
         private Tuple<string, double?> Resolve(IEnumerable<double> enumerable)
         {
             double doubleTol = 1e-6;
-            var list = enumerable.ToList();
+            var list = enumerable.Distinct().ToList();
```

Comment:
it does display all the options, though. It displays all proteins that those peptides could have come from. It just removes duplicates. It gets especially confusing when a PSM is ambiguous and one option is a shared peptide. then you have peptides 1,2 with proteins A,B,C - there is no indication of which peptide comes from what protein anyway